### ISSUE TITLE
Navigator cannot Activate Environment in ZSH

### ISSUE THREAD

**Issue by aisync** [ContinuumIO/anaconda-issues#11608]
### Actual Behavior

**Activation via navigator returns:**
```
/Users/name/.anaconda/navigator/a.tool ; exit;
 name@nameMacBook-Pro  ~  /Users/name/.anaconda/navigator/a.tool ; exit;
/Users/name/.anaconda/navigator/a.tool: line 1: syntax error near unexpected token `('
/Users/name/.anaconda/navigator/a.tool: line 1: `bash --init-file <(echo "source activate /usr/local/anaconda3/envs/myenv;")'

[Process completed]
```

1-line contents of `/Users/name/.anaconda/navigator/a.tool` is

`bash --init-file <(echo "source activate /Users/name/opt/anaconda3;")`

---
**Activating environment from any shell:**
This separate issue coincides with this one however it's not clear to me if entirely related. 

`conda activate r_env` returns:
```
/Users/name/opt/anaconda3/envs/r_env/etc/conda/activate.d/java_home.sh:6: = not found
```
Similarly, the content of which is as follows:
```
export CONDA_BACKUP_JAVA_HOME="${JAVA_HOME}"
export JAVA_HOME="${CONDA_PREFIX}"

export CONDA_BACKUP_JAVA_LD_LIBRARY_PATH="${JAVA_LD_LIBRARY_PATH}"

if [ $(uname) == Darwin ]; then
  export JAVA_LD_LIBRARY_PATH="${JAVA_HOME}"/jre/lib/server
else
  if [ $(uname -m) == x86_64 ]; then
    export JAVA_LD_LIBRARY_PATH="${JAVA_HOME}"/jre/lib/amd64/server
  else
    export JAVA_LD_LIBRARY_PATH="${JAVA_HOME}"/jre/lib/i386/server
  fi
fi
```

[Reference to the second issue & suggested solution that I didn't understand](https://stackoverflow.com/questions/60183391/anaconda-i-cannot-switch-environment-without-error-java-home-sh-not-found?noredirect=1#comment106449433_60183391)

_There seem to be various issue with navigator revolving around proper environment setup._ 
#11334 
#11585 
#11123 

Any help with this would be tremendously appreciated.

##### `conda info`

<details>

```
     active environment : base
    active env location : /Users/name/opt/anaconda3
            shell level : 1
       user config file : /Users/name/.condarc
 populated config files : /Users/name/.condarc
          conda version : 4.8.2
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       virtual packages : __osx=10.15.3
       base environment : /Users/name/opt/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/name/opt/anaconda3/pkgs
                          /Users/name/.conda/pkgs
       envs directories : /Users/name/opt/anaconda3/envs
                          /Users/name/.conda/envs
               platform : osx-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.4 Darwin/19.3.0 OSX/10.15.3
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

</details>

##### `conda list --show-channel-urls`

<details>

```
PASTE OUTPUT HERE:

```

</details>

**Comment by huangxingnan1994** [comment]
I've got the same issue as well. Do you have any resolution now?

**Comment by aisync** [comment]
No fix yet unfortunately. For me It seems to be an issue on new conda installs. My older iMac works perfectly fine, but newer Macbook presents the error every time. Do you have a recent install?

**Comment by forrestwaters** [comment]
The first issue listed here is fixed in Navigator 1.9.12. Please try upgrading with:
```
conda update anaconda-navigator
```
The second issue described was due to bash-ism's in our activate scripts for openjdk. New packages are currently being uploaded that should address that.
